index.html

```diff
@@ -30,19 +31,59 @@
     <br />
     <br />
     <div class="bttn-container">
-      <button class="bttn-1">
+      <button class="bttn-1" id="about-bttn">
         <div class="overlap-group">
           <div class="buttn-1-text">sobre</div>
-          <!-- <div class="popup" onclick="myFunction()">
-            <span class="popuptext" id="myPopup">Popup text...</span> -->
         </div>
       </button>
+      <div id="aboutModal" class="modal">
+        <!-- Modal content -->
+        <div class="modal-content">
+          <span class="close" id="closeAbout">&times;</span>
```

Comment:
Same from comment above.

```suggestion
          <span class="close" id="close-about">&times;</span>
```